packages/unique/src/unqiueModule.ts

```diff
@@ -0,0 +1,23 @@
+import {IConstraintModule, ConstraintValue, ConstraintExecuter, ConstraintConfiguration, IRuntimeContext, IValidator} from '@configurable-validator/core'
+
+
+export class UnqiueModule implements IConstraintModule {
```

Comment:
didn't understand the implementation

Reply:
unique validates that the given collection has only unique items by a given key\property path inside each item.

Reply:
This wasn't actually implemented, sorry for that.